multicluster/config/overlays/leader-ns/coverage/manager_command_patch_coverage.yaml

```diff
@@ -8,6 +8,8 @@ spec:
     spec:
       containers:
         - command: ["/bin/sh"]
-          args: ["-c", "/antrea-mc-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-mc-controller.cov.out leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
-          name: antrea-mc-controller
+        - args:
+          - antrea-mc-controller-coverage
```

Comment:
What's the value of  `-test.coverprofile`  after this change?

Reply:
We don't use that flag anymore. The instrumented binary will generate covmeta and covcounter files. They will be moved to our local coverage directory and then converted to cov.out files using go tool covdata.

Reply:
I think this will impact the way to upload the coverage to codecov, please check here https://github.com/antrea-io/antrea/blob/a8d78bca87f200394e18db50e8416089ce1f9e36/ci/jenkins/test-mc.sh#L469 and correct it correspondingly.
Please double check and verify that all new generated coverage files can be uploaded to codecov site correctly.